test/functional/test_site_convert/expected/contents/topic1.html

```diff
@@ -61,10 +61,11 @@
         </div>
       </nav>
       <div id="content-wrapper">
-        <br>
+        <br cwf="/home/crphang/markbind/test/functional/test_site_convert/non_markbind_site/contents/topic1.md">
```

Comment:
A quick review in passing... seems like this should not be appearing as part of the output?

Reply:
Yes. I realized that so reverted "ready for review" :sweat:. The ordering of how when the parser is invoke matter. I changed it a little and this happened, even though functionally it works.

I'll see how far I could work around the parser on this diff. If things would start to look really messy, I would see if some refactor of the parser have to be done to make this work better.